### PR TITLE
Detailed the description to enable easy setup

### PR DESCRIPTION
**Pull request by helios1101**
For beginners, sometimes it becomes important to lay the complete path to follow, hence detailed step comes to rescue.

@tbedford you may have a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,18 @@ Minimal doc product is a small repo that is designed to give you an starting poi
...
 
-1. Make sure you have permissions for https://rubygems.pkg.github.com/nexmo. Instructions are [here](https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-rubygems-for-use-with-github-packages)
-2. Clone this repo.
-3. Change into cloned directory.
-4. Check the `Gemfile` for the correct Ruby version.
-5. Select the correct version of Ruby, for example:
+`NOTE: Commands are to be executed on Linux, Mac, and Windows(using Powershell)`
 
+1. Make sure you have permissions for https://rubygems.pkg.github.com/nexmo. Instructions are [here](https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-rubygems-for-use-with-github-packages)
+2. Clone the repo by running the command
```
Comment (tbedford) [inline README.md]: If breaking like this use colon at end of sentence and then have a blank line before the code block.

File: README.md [hunk 2]
```diff
@@ -10,12 +10,18 @@ Minimal doc product is a small repo that is designed to give you an starting poi
...
+```sh
+$ git clone https://github.com/Your_Username/minimal-doc-product.git
```
Comment (tbedford) [inline README.md]: Your_Username -> nexmo-community

File: README.md [hunk 3]
```diff
@@ -10,12 +10,18 @@ Minimal doc product is a small repo that is designed to give you an starting poi
...
+```
+3. Go inside the cloned directory by command
```
Comment (tbedford) [inline README.md]: If breaking like this use colon at end of sentence and then have a blank line before the code block.

File: README.md [hunk 4]
```diff
@@ -10,12 +10,18 @@ Minimal doc product is a small repo that is designed to give you an starting poi
...
+```sh
+$ cd minimal-doc-product
+```
+4. Check the `Gemfile` for the correct Ruby version and select the correct version, for example:
```
Comment (tbedford) [inline README.md:29]: If breaking like this use colon at end of sentence and then have a blank line before the code block.

File: README.md [hunk 5]
```diff
@@ -35,7 +41,7 @@ OAS_PATH="`pwd`/_open_api/api_specs/definitions" bundle exec nexmo-developer --d
 
 ### To use as the starting point for your own product
 
-If you want to use this repo as the starting point for your own doc product you can **fork** this repo on GitHub.
+If you want to use this repo as the starting point for your own doc product you can **fork** this repo on GitHub by clicking on the <kbd><b>Fork</b></kbd></a> button (Top-Left).
```
Comment (tbedford) [inline README.md]: Fork button is top right.

**Review by tbedford — CHANGES_REQUESTED** [review]
The target audience is not really beginners, but thanks for your suggestions. I have requested a few changes if you get time.

File: README.md (hunk 1 above)
Comment (helios1101) [reply README.md]: Sure, updated.

File: README.md (hunk 2 above)
Comment (helios1101) [reply README.md]: Sure, updated.

File: README.md (hunk 3 above)
Comment (helios1101) [reply README.md]: Sure, thanks.
Updated.

File: README.md (hunk 4 above)
Comment (helios1101) [reply README.md:29]: Sure, thanks.
Updated.

File: README.md (hunk 5 above)
Comment (helios1101) [reply README.md]: Sure, thanks.
Updated.

**Comment by helios1101** [comment]
Thanks @tbedford for your valuable feedbacks.
You may have a look.

**Review by tbedford — APPROVED** [review]
Thank you!

**Pull request merged**